hlx_statics/styles/styles.css

```diff
@@ -11,6 +11,7 @@
  */
 
 @import url('./spectrum/spectrum.min.css');
+@import url('./prism.css');
```

Comment:
```suggestion
```

Reply:
For lighthouse score, we're only importing prism if needed: https://github.com/AdobeDocs/adp-devsite/blob/40d88026fb0e0488bd428ac3c3eab4967a9dc18e/hlx_statics/scripts/delayed.js#L26